pkg/actor/director.go

```diff
@@ -78,6 +87,13 @@ func (cd *clusterDirector) GetActorsToExecute(cluster *resource.Cluster) []Actor
 
 	var actorsToExecute []Actor
 
+	// we need to ensure the SA exists early on since the version checker job uses it which happens before the deploy.
+	ctx := context.Background()
+	serviceAccounts := cd.clientset.CoreV1().ServiceAccounts(cluster.Namespace())
+	if _, err := serviceAccounts.Get(ctx, cluster.ServiceAccountName(), metav1.GetOptions{}); kube.IsNotFound(err) {
```

Comment:
sweet, this should make it easy to merge in with the one-actor PR (whichever comes first)